Data

```diff
@@ -1 +1 @@
-C:/Users/Brandyn/Documents/GitHub/EchoPro_data/2019_consolidated_files
\ No newline at end of file
+C:/Users/Brandyn/Documents/GitHub/EchoPro_data/2019_consolidated_files
```

Comment:
Probably a good idea to not push the symbolic link `Data`. If this is used somewhere (though it really shouldn't be in the long run), we can add how to set it up to the README.